content/BatchResponseContentCollection.md

```diff
@@ -0,0 +1,24 @@
+# BatchResponseContentCollection
+
+## Objectives
+
+Provide component to simplify the processing of multiple batch responses made via `BatchRequestContentCollection`.
+
+## Requirements
+
+See [constraints](ContentArchitecturalConstraints.md) related to all content objects.
+
+- Hold multiple batch response objects (i.e. `BatchResponseContent`) made from a collection of batch request objects(i.e. `BatchRequestContentCollection`)
+- Allow users to enumerate collection of individual response objects
```

Comment:
How are we managing the async nature of executing 5 batches of 20 queries? Do we block until all 5 batches are back? Async Iterators?

Reply:
Current implementations wait for all the five batches to complete and come back. We could investigate using Async Iterators but I'm not too sure about this when we consider cross language consistency.